Stratis.SmartContracts/UInt256.cs

```diff
@@ -23,7 +23,7 @@ public static UInt256 Parse(string str)
             return new UInt256(str);
         }
 
-        public UInt256(BigInteger value)
+        private UInt256(BigInteger value)
```

Comment:
Internal is better probably based on code changes

Reply:
See above.